.github/workflows/prefer-typescript.yml

```diff
@@ -2,6 +2,8 @@ name: Prefer TypeScript
 
 on:
   pull_request:
+    paths-ignore:
+      - 'docs/**'
```

Comment:
Don't we want to prefer TypeScript in docs?

Reply:
Yes, but I figured we didn't want it muddying our metrics tracked with that action

Reply:
In any case, undid this one, thanks.